exmg-form-styles.ts

```diff
@@ -0,0 +1,64 @@
+import {html} from 'lit-element';
+import {sharedButtonStyles} from '@exmg/exmg-cms-styles/exmg-cms-button-styles.js';
+
+export const exmgFormStyles = html`
```

Comment:
Use css tag instead of html: https://lit-element.polymer-project.org/guide/styles

Reply:
done